CONTRIBUTING.md

```diff
@@ -1,11 +1,76 @@
 # Contributing
 
-## Style
+## System Dependencies
 
-More to come soon.
+### Node/npm
+
+This projects uses version `2` of the package-lock.json file. Version `2` requires `npm` 7+. npm `8` comes prebundled with Node `16`.
+
+## Install project dependencies
+
+### Code
+
+* Clone the repo
+
+### Install project dependencies
+
+`npm install`
+
+This installs dependencies from `package-lock.json`.
+
+### Contentful environment variables
+
+Generate keys in Contentful. See **Settings > API keys**.
+
+Copy the `.env.local.example` file in this directory to `.env.local`:
+
+```bash
+cp .env.local.example .env.local
+```
+
+Then set each variable on `.env.local`:
+
+- `CONTENTFUL_SPACE_ID` should be the **Space ID** field of your API Key
+- `CONTENTFUL_ACCESS_TOKEN` should be the **[Content Delivery API](https://www.contentful.com/developers/docs/references/content-delivery-api/) - access token** field of your API key
+- `CONTENTFUL_PREVIEW_ACCESS_TOKEN` should be the **[Content Preview API](https://www.contentful.com/developers/docs/references/content-preview-api/) - access token** field of your API key
+- `CONTENTFUL_PREVIEW_SECRET` should be any value you want. It must be URL friendly as the dashboard will send it as a query parameter to enable preview mode
```

Comment:
Should `CONTENTFUL_ENVIRONMENT` be added here? Or replace `CONTENTFUL_PREVIEW_SECRET`.

Reply:
Ah yes, good catch! I'll get that updated.